activesupport/lib/active_support/core_ext/module/attribute_accessors_per_thread.rb

```diff
@@ -111,7 +111,7 @@ def #{sym}=(obj)
   #   class Customer < Account
   #   end
   #
-  #   Customer.user = "CHH"
+  #   Customer.user = "DHH"
```

Comment:
Ahhh! That makes sense. The "CH" looks a lot like "DH" in monotype fonts, I figured it had to be a typo.